lib/sparrow/h2_worker/config.ex

```diff
@@ -18,7 +18,10 @@ defmodule Sparrow.H2Worker.Config do
           reconnect_attempts: pos_integer,
           backoff_initial_delay: pos_integer,
           backoff_max_delay: pos_integer,
-          backoff_base: pos_integer
+          backoff_base: pos_integer,
+          pool_type: Sparrow.PoolsWarden.pool_type(),
+          pool_name: atom,
+          tags: [atom]
```

Comment:
I would name this `pool_tags` also, as there is no other way of knowing what tags those are, being in context of a `worker`.